### PR TITLE
@releng - update version numbers after the release of version 1.9.0

### PR DESCRIPTION
**Pull request by LSantha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,9 @@ The latest version of the AEM CIF Core Components, require the below minimum sys
 
 | CIF Core Components | AEM as a Cloud Service | AEM 6.5 | AEM 6.4 | Magento | Java  |
 | ------------------- | ---------------------- | ------- | ------- | ------- | ----- |
-| 1.8.0               | Continual              | 6.5.7   | 6.4.4.0 | 2.4.0   | 8, 11 |
+| 1.9.0               | Continual              | 6.5.7   | 6.4.4.0 | 2.4.0   | 8, 11 |
+
+Version 1.9.0 is also compatible with Magento 2.4.1 and 2.4.2 .
```
Comment (mhaack) [inline README.md:49]: In the past we just added the lastet version in the table.

Comment (LSantha) [reply README.md:49]: Above the table it writes minimal required versions which would mean that v. 1.9.0 is not compatible with anything older than Magento 2.4.2.

Comment (mhaack) [reply README.md:49]: Maybe the lets use `2.4.0 - 2.4.2` similar like we do for Java column. So it is clear on the first sight.

Comment (LSantha) [reply README.md]: OK, done.

**Review by herzog31 — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Pull request merged**